intrastat_product/models/intrastat_product_declaration.py

```diff
@@ -1131,67 +1065,66 @@ class IntrastatProductComputationLine(models.Model):
         "at the pro-rata of the amount of each invoice line.",
     )
     transaction_id = fields.Many2one(
-        "intrastat.transaction", string="Intrastat Transaction"
+        "intrastat.transaction",
+        string="Intrastat Transaction",
     )
     region_id = fields.Many2one("intrastat.region", string="Intrastat Region")
-    # product_origin_country_id is replaced by product_origin_country_code
-    # this field should be dropped once the localisation modules have been
-    # adapted accordingly
+    # Note that, in l10n_fr_intrastat_product and maybe in other localization modules
+    # region_id is left empty and Odoo writes directly in region_code
+    region_code = fields.Char()
```

Comment:
Perhaps another option would be to make the compute store=True field to avoid the onchange and be able to overwrite it in other modules.

Reply:
No, because, in l10n_fr_intrastat_product, I don't use the object "intrastat.region", because I already have another object for that in the french localisation. That's why I want to have the char field region_code.